ZendeskSDK.podspec

```diff
@@ -17,36 +17,36 @@ Pod::Spec.new do |s|
   }
   s.author       = 'Zendesk'
   s.source       = { :git => "https://github.com/zendesk/zendesk_sdk_ios.git", :tag => s.version }
-  s.platform     = :ios, '9.0'
+  s.platform     = :ios, '9.3'
   s.requires_arc = true
   s.frameworks = 'MobileCoreServices', 'SystemConfiguration', 'Security', 'MessageUI'
-  s.swift_version = '5.0'
+  s.swift_version = '5.0.1'
   s.xcconfig = { 'FRAMEWORK_SEARCH_PATHS' => '$(inherited)', 'CLANG_ENABLE_MODULES' => 'YES', 'ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES' => 'YES' }
```

Comment:
Can this be changed to 'NO' since 5.0.1

Reply:
I think it might be needed still. The swift library is only embedded on iOS 12.2 and higher so I think for other OS' it will not build.